graphql/resolvers.go

```diff
@@ -1160,26 +1160,14 @@ func (r *queryResolver) PatchTasks(ctx context.Context, patchID string, sorts []
 }
 
 func (r *queryResolver) TaskTests(ctx context.Context, taskID string, execution *int, sortCategory *TestSortCategory, sortDirection *SortDirection, page *int, limit *int, testName *string, statuses []string) (*TaskTestResult, error) {
-	dbTask, err := task.FindByIdExecution(taskID, execution)
-	if dbTask == nil || err != nil {
-		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find task with id %s", taskID))
-	}
-	baseTask, err := dbTask.FindTaskOnBaseCommit()
-	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding base task with id %s: %s", taskID, err))
-	}
-
-	var taskExecution int
-	taskExecution = dbTask.Execution
-
-	baseTestStatusMap := make(map[string]string)
-	if baseTask != nil {
-		baseTestResults, _ := r.sc.FindTestsByTaskId(baseTask.Id, "", "", "", 0, taskExecution)
-		for _, t := range baseTestResults {
-			baseTestStatusMap[t.TestFile] = t.Status
-		}
-	}
-
+	opts := apimodels.GetCedarTestResultsOptions{
+		BaseURL:   evergreen.GetEnvironment().Settings().Cedar.BaseURL,
+		TaskID:    taskID,
+		Execution: *execution,
+	}
+	cedarTestResults, _ := apimodels.GetCedarTestResults(ctx, opts)
```

Comment:
we should handle the error

Reply:
Cedar Returns an error if it doesn't find a test log in that case we fall back to fetch the logs from evergreen. I think its safe to ignore this error since all it means is that cedar doesn't have any test logs for a task.